README.md

```diff
@@ -1,634 +1,1855 @@
-- **No Superfluous "dApp" Tokens** - ADA is all you need to pay for script/TX fees. 
-- **Democratic Upgradability** - users choose if/when to use new contracts.
+## Motivation
 
+A healthy credit-debt market is a vital (and often the largest) component of a thriving economy. A
+healthy market is one in which prices reflect underlying reality as quickly and faithfully (with as
+little friction) as possible. For the sake of avoiding a whole treatise on economics, suffice to say
+that *the best way to achieve fast and frictionless price discovery is through the aggregation of
+maximally expressive individual sentiments.* This is especially true for credit-debt markets, where
+money itself is the asset, priced via interest rates. Futhermore, as with all DeFi protocols, any
+credit-debt market that seeks to be an alternative to the current global financial system will
+likely be perceived as a threat by those in positions of high power, so censorship resistance is an
+essential feature. 
+
+Unfortunately, contemporary lending/borrowering DApps on Cardano are implemented in ways that cannot
+possibly create a true trustless and decentralized credit-debt market. To fully appreciate this, it
+is first important to understand the deficiencies of the status quo:
+
+### Oracles
+
+DApps that rely on off-chain information feeds are subject to the integrity of the underlying oracle
+network. It is pretty easy to find examples of faulty oracle feeds causing DeFi issues that resulted
+in users losing money. Not only that, but in many cases, *oracles are simply not necessary*. As long
+as users are able to negotiate the terms *before* initiating the loan, there is no need for
+liquidations, and therefore, no need for oracles. In TradFi, liquidations are usually only used
+with *margin loans* which are typically short-term loans and/or are generally only used to buy more
+securities (ie, speculate on markets). In a typical economy, most loans are *not* margin loans; you
+wouldn't use a margin loan to start a business.
+
+A lot of people believe oracles are required for DeFi because of the following:
+
+> What if the value of the colleral falls 6 months into the loan?! Shouldn't Alice be forced to put
+> up more collateral?
+
+No, she shouldn't. Bob, her lender, knew the loan was going to be a year long loan. If he was
+concerned about the value of the collateral falling during the life of the loan, he could have asked
+for more collateral and/or asked for different collateral. The fact that cryptocurrency markets can
+be extremely volatile does not matter. Volatile assets also exist in TradFi (eg, small cap stocks,
+emerging market assets, etc), and this is exactly how TradFi manages the risk. If the collateral
+ever falls too low, relative to the outstanding loan, **it is the lender's fault**. The borrower
```

Comment:
TradFi uses margin loans when stocks/shares are the collateral.

Reply:
I agree that is *usually* the case. I do not personally work on wall street so I don't have concrete evidence, but I believe this is not always true. I think it depends on the volatility of stocks/shares. A small cap stock will likely either be rejected as collateral or they will want to use margin loans. Meanwhile a large cap stock with low volatility may be accepted as collateral without margin loans (also accounting for the loan's duration). I don't think this approach is open to the average person, but millionaires and billionaires likely do get deals like this. 

I can change the sentence to: "and this is one of the methods TradFi uses to manage the risk."